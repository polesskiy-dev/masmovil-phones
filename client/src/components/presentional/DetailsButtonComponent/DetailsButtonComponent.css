.details-button {
    background-color: var(--details-button-bg-color);
    color: var(--details-button-color);

    padding: 6px 12px;

    font-size: 14px;
    line-height: 1.42857;
    text-transform: uppercase;
    font-weight: 800;

    border-radius: 0;
    border: 1px solid transparent;

    text-decoration: none;
    cursor: pointer;
    text-align: center;
    vertical-align: middle;

    transition: all .35s cubic-bezier(.25,.46,.45,.94);
}