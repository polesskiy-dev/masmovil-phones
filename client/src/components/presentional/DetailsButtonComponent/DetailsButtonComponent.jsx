import React from 'react';
import { FormattedMessage } from 'react-intl';

import './DetailsButtonComponent.css';

const DetailsButtonComponent = () => (
  <button className="details-button">
    <FormattedMessage id="Buttons.detailsButtonCaption" />
  </button>
);

export default DetailsButtonComponent;
