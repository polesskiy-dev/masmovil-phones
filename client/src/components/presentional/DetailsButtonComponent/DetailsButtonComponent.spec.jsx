import React from 'react';
import renderer from 'react-test-renderer';
import _ from 'lodash/fp';

import withDefaultIntl from '../../../helpers/with-default-intl';
import DetailsButtonComponent from './DetailsButtonComponent';

it('<PhoneDetailComponent/ > renders correctly with all required fields', () => {
  const tree = renderer
    .create(withDefaultIntl(<DetailsButtonComponent onClick={_.noop} />))
    .toJSON();
  expect(tree).toMatchSnapshot();
});
