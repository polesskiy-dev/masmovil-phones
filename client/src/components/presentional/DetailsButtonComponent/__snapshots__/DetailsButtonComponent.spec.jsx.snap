// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhoneDetailComponent/ > renders correctly with all required fields 1`] = `
<button
  className="details-button"
  onClick={[Function]}
>
  <span>
    Details
  </span>
</button>
`;
